internal/rbd/rbd_healer.go

```diff
@@ -95,11 +92,11 @@ func formatStagingTargetPath(c *k8s.Clientset, pv *v1.PersistentVolume, stagingP
 	return targetPath, nil
 }
 
-func callNodeStageVolume(ns *NodeServer, c *k8s.Clientset, pv *v1.PersistentVolume, stagingPath string) error {
+func callNodeStageVolume(ns *NodeServer, pv *v1.PersistentVolume, stagingPath string) error {
```

Comment:
Any function that gets the `NodeServer` as parameter is a good candidate to become a function of the `NodeServer struct`.

This is something that you may want to cleanup in a follow-up PR (or separate commit).